cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -315,14 +333,21 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
         -- to stdout
         -- TODO: deserialize to a Haskell value when
         -- https://github.com/IntersectMBO/cardano-cli/issues/606 is tackled
-        dreps :: Aeson.Value <- H.noteShowM $ execCliStdoutToJson execConfig [ eraName, "query", "drep-state", "--all-dreps" ]
-        assertArrayOfSize dreps 3
+        dreps <- H.noteShowM $ execCliStdoutToJson execConfig [ eraName, "query", "drep-state", "--all-dreps", "--include-stake"]
+        let drepsRedacted = redactJsonFields (Map.fromList [("keyHash", "<redacted>")]) dreps
```

Comment:
You could move the mapping of `"<redacted>"` to inside `redactJsonFields` unless you have plans to change the update the value to something other than `"<redacted>"`.

Reply:
I think it could be useful in the future to use other terms than `"<redacted>"`. I mean it could be "<value greater than foo>" for example, to give a bit more insight.

So I'm leaving it like this, because the overhead is minimal.